apps/namadillo/src/App/Masp/MaspShield.tsx

```diff
@@ -204,7 +204,7 @@ export const MaspShield: React.FC = () => {
             />
           </motion.div>
         )}
-        {currentStep > 0 && (
+        {currentStep > 0 && transaction?.currentStep && (
```

Comment:
Should we rename this to currentStepIndex as well? Just to be equal do MaspUnshiled?